my-code/src/components/Movies.js

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect, useState } from 'react'
+import { FetchMovies } from "./FetchMovies";
+
+export function Movies() {
+
+    const [query, setQuery] = useState("");
+    const [disabled, setDisabled] = useState(true);
+    const [search, setSearch] = useState(false);
+
+    let apiKey = "798e7a67";
+
+    let min = 3;
+
+    function queryMovie() {
+        let q = document.getElementById("q");
+        let results = document.getElementById("results");
+        setQuery(q.value);
+        setSearch(true);
+        results.innerHTML = `Results for ${q.value}:`;
+        document.title = `${q.value} | MOVIES SEARCH`;
+
+    }
+
+    useEffect(() => {
+        document.getElementById("q").focus();
+    })
```

Comment:
It's important to set the dependencies on `useEffect`.
https://dev.to/spukas/4-ways-to-useeffect-pf6